packages/desktop/views/dashboard/collectibles/views/DetailsView.svelte

```diff
@@ -10,48 +10,19 @@
         Text,
     } from 'shared/components'
     import { localize } from '@core/i18n'
+    import { selectedNftId } from '../stores/selected-nft.store'
+    import { getNftByIdFromAllAccountNfts, INft } from '@core/nfts'
+    import { selectedAccountIndex } from '@core/account'
+    import { truncateString } from '@core/utils'
 
-    export let id: string = 'vt7ryeasd6f5hfh689w'
-    export let name: string = 'IOTABOTS #5'
-    export let collection: string = 'IOTABOTS'
-    export let collectionUrl: string = 'https://iotabots.io'
-    export let type: string = 'IRC27'
-    export let storageDeposit: string = '0.5 MI'
-    export let attributes: { title: string; value: string | number | boolean }[] = [
-        {
-            title: 'Background',
-            value: 'Purple',
-        },
-        {
-            title: 'Element',
-            value: 'Water',
-        },
-        {
-            title: 'Attack',
-            value: 150,
-        },
-        {
-            title: 'Super Long Attribute',
-            value: 2000,
-        },
-        {
-            title: 'Another Attribute',
-            value: 1000,
-        },
+    const nft: INft = getNftByIdFromAllAccountNfts($selectedAccountIndex, $selectedNftId)
 
-        {
-            title: 'Long Attribute',
-            value: 2000,
-        },
-        {
-            title: 'Another Attribute',
-            value: 1000,
-        },
-    ]
+    const { id, name } = nft
+    const { type, collectionName, attributes } = nft.parsedMetadata
```

Comment:
`parsedMetadata` is an optional parameter. If it is not a field on the nft it will result in an error. We could give default values in that case like:
```
const { type, collectionName, attributes } = nft.parsedMetadata || { undefined, 'Not Found', undefined };
```
Or we could just make it the below so we don't have an error 
```
const { type, collectionName, attributes } = nft.parsedMetadata || {}
```

Reply:
good point